### DISCUSSION TITLE
Existe o canal morangos com açucar na tvi player, e também Existem os canais de desporto da RTP no RTP player, valerá a pena colocar na lista?

### DISCUSSION THREAD

**Discussion by cportugalr** [LITUATUI/M3UPT#50] [General]
Existe o canal morangos com açúcar na tvi player, e também existem canais de desporto da RTP no RTP player, valerá a pena colocar na lista?

**Comment by LITUATUI** [comment]
Os canais de desporto talvez, se tiverem emissões frequentes. Vou ficar atento.

**Comment by LITUATUI** [comment]
Já adicionei o canal MCA.

**Comment by cportugalr** [comment]


**Reply by thomraider12** [reply]
Penso que sejam todos legais, sendo que estão disponíveis no YouTube, ou sites oficiais do canal.
